tools/transformer/src/transformations/fields.js

```diff
@@ -0,0 +1,29 @@
+const fieldTransformations = require("../field.transformations.json");
+
+/**
+ * Makes specified fields optional in their respective schemas
+ * @param {*} api OpenAPI JSON File
+ * @returns OpenAPI JSON File
+ */
+function applyFieldTransformations(api) {
+  const { optionalFields } = fieldTransformations;
```

Comment:
find it strange why we add this level of indirection with the file, instead of just having the array here